configure.ac

```diff
@@ -86,7 +86,7 @@ AC_PATH_TOOL(RANLIB, ranlib)
 AC_PATH_TOOL(STRIP, strip)
 AC_PATH_TOOL(GCOV, gcov)
 AC_PATH_PROG(LCOV, lcov)
-AC_PATH_PROG(PYTHON, python)
+AC_PATH_PROG(PYTHON, python3)
```

Comment:
So py3 become an hard req? not a big problem just want to make sure I understood correctly.

Reply:
Yes, unfortunately, kind of. The problem is that there is just one variable in the autotools that deals with python. I have not found a way how to keep parallel python variables and use their macros. Of course, one could rewrite the macros, but then that's also a maintenance problem. I think it is easiest to require python3. AFAIK, the current tests already run on py3, or don't they?

Reply:
yeah, in fact is not a big deal at all. we already enforce the use of py3 for the functional tests.